upgrade.php

```diff
@@ -95,7 +95,7 @@
 	);
 
 	// reset cache to have latest translations available during upgrade
-	elgg_reset_system_cache();
+	elgg_flush_caches();
```

Comment:
no need to flush all... just reset system cache, as the comment states it is only needed for the translations